README.md

```diff
@@ -265,12 +265,31 @@ showLocation({
 });
 ```
 
+Alternatively you can specify the `address` field and leave the latitude and longitude properties as empty strings 
+
+```js
+import {showLocation} from 'react-native-map-link';
+
+showLocation({
```

Comment:
suggestion: could we do a simpler example here, without all of the optional fields, to better illustrate this alternative usage?

Reply:
Awesome. I made that change. 👍